lib/govuk_content_models/presentation_toggles.rb

```diff
@@ -25,14 +25,38 @@ def organ_donor_registration_url
   end
 
   def organ_donor_registration_key
-    presentation_toggles['organ_donor_registration']
+    presentation_toggles['organ_donor_registration'] || self.class.default_presentation_toggles['organ_donor_registration']
```

Comment:
I reckon it'd be worth explaining in the commit message why we've had to do this - it seems weird because mongoid already uses a default for the field if it's missing.  The problem we're getting around is that our default value changed with this commit and so existing records have the old value that is missing the extra keys we've added.